packages/web-components/@sfx/autocomplete/src/autocomplete.ts

```diff
@@ -22,6 +22,10 @@ export default class Autocomplete extends LitElement {
    * The text to use in the header.
    */
   @property({ type: String, reflect: true }) caption: string = '';
+  /**
+   * The optional search box ID this will check in events.
+   */
+  @property({ type: String }) searchbox: string = undefined;
```

Comment:
I made it `undefined` because that's what comes back in an event from the logic layer if the no searchbox was sent. I could make this `''`, but it adds another step for the equality check in `receivedResults()`.

Reply:
I'm okay with that. It makes sense to me that we'd either have a non-empty `searchbox` property (ie. they set it to something) or we'd have it not be defined.